common/locales/en/quests.json

```diff
@@ -48,6 +48,8 @@
     "leaveQuest": "Leave Quest",
     "sureLeave": "Are you sure you want to leave the active quest? All your quest progress will be lost.",
     "questOwner": "Quest Owner",
+    "questTaskDamage": "pending damage to boss",
```

Comment:
You should pass parameters into these strings instead of manually concatenating them with numbers wherever the strings are used.